cli/popper/cli.py

```diff
@@ -1,7 +1,7 @@
 import os
 import sys
 import click
-
+import popper
```

Comment:
Since this file is already in the `popper` package, importing it a package from itself it's kind of confusing. The "pythonic" way is to use [relative imports](https://docs.python.org/3/tutorial/modules.html#intra-package-references). In this case, this could work:

```python
from . import __version__ as popper_version
```